coala_quickstart/green_mode/green_mode_core.py

```diff
@@ -11,9 +11,6 @@
     initialize_project_data,
     run_quickstartbear,
     )
-from coala_quickstart.green_mode.filename_operations import (
```

Comment:
I don't think the issue was about removing this import. This is what the issue says :point_down: 

> coala_quickstart/green_mode/green_mode_core.py :: green_mode contains a delayed import of check_filename_prefix_postfix which should be removed as it shadows an identical import in the module scope.

the keyword here is delayed, you need to delete just the delayed import if you delete the first import then we won't be able to call this function on this [line](https://github.com/coala/coala-quickstart/pull/330/files#diff-97cf9e66d91e4241e9a7aa335c9d311bR66)

Please make the correct changes @Manas-kashyap and if you don't understand the issue don't hesitate to ask us before submitting a PR :wink: